build/build.xml

```diff
@@ -461,6 +462,10 @@
 		<build-deploy-addon name="todo" />
 	</target>
 
+	<target name="deploy-fieldenumeration" description="Provides blacklist characters">
```

Comment:
Should be something like:
> Deploys the Field Enumeration extension
or something like that, look at other entries for examples.